app/serializers/invoice_item_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class InvoiceItemSerializer < ActiveModel::Serializer
+  attributes :id, :item_id, :invoice_id, :quantity, :unit_price, :created_at, :updated_at
```

Comment:
@akintner i like that you're using serializers 👍 is this any different from the data in the db though?